tests/functional_tests/serialization_test.py

```diff
@@ -38,10 +38,7 @@ def electron_function(x):
 
 
 @etron
```

Comment:
This looks like a typo

Reply:
This has the same reason as mentioned below, its for a test (I think related to importing) and was introduced before this PR.